### PR TITLE
Improve website navigation bar

### LINKED ISSUE #13796: Move 'modules' link to LH side of website?
Currently the link to "modules" is on the far RH side of the website:
Screenshot 2019-03-29 at 11 23 02

Often I find myself clicking "API", expecting to have a list of modules, and it takes me a while to figure out that what I actually want is the modules link, but it's a bit hidden on the RH side. Would there be any objections to moving it next to the "API" link on the LH side?

### PR DESCRIPTION
**Pull request by dstansby**
Fixes #13796 - I've also capitalised the entries.

### CODE REVIEW

**Comment by dstansby** [comment]
https://19829-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/index.html

**Comment by timhoffm** [comment]
Modules is now duplicate

https://19829-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/index.html

The right one comes somehow through the sphinx template mechanism.

Note that we'll probably have to rework the sphinx template anyway (either due to sphinx 2.0 #13591 or as part of a more complete documentation structure overhaul https://github.com/matplotlib/matplotlib/issues/13796#issuecomment-478284666). So I personally, wouldn't bother too much with this single link now.

**Comment by dstansby** [comment]
🤷‍♂️ I think it's an improvement, and having an obvious link but a duplicate is better than not having an obvious link.

**Comment by timhoffm** [comment]
Hm, I don't think having two links with the same name but different targets adds to the clarity. But I don't care too much since I assume that this will be reworked anyway in the near future.

**Comment by NelleV** [comment]
I'm not a big fan of the term "module." IMO, it's unclear what it is and should be folded into the API menu.

**Comment by NelleV** [comment]
(I had to click on module in order to understand what the link was about)

File: doc/api/api_overview.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ Further reading:
   plotting functions.
 - Most of the :ref:`examples <examples-index>` use the object-oriented approach
   (except for the pyplot section)
-- The list of :doc:`matplotlib modules </api/index>`.
+- The list of :doc:`Mwatplotlib modules </api/index>`.
```
Comment (timhoffm) [inline doc/api/api_overview.rst]:
```suggestion
- The list of :doc:`Matplotlib modules </api/index>`.
```

**Comment by anntzer** [comment]
Perhaps "submodules"(/"subpackages")?

**Comment by dstansby** [comment]
I think this is covered by #14176 now, so I'm going to close.

**Pull request closed without merging**